server/chat.js

```diff
@@ -272,6 +272,7 @@ Chat.nicknamefilters = [];
  * @param {boolean} [forStatus]
  */
 Chat.nicknamefilter = function (nickname, user, forStatus = false) {
+	if (forStatus) nickname = nickname.replace(/\|/g, '');
```

Comment:
Nicknames shouldn't contain `\|` either.

Reply:
(It's already enforced elsewhere, but there's no harm in also enforcing it here.)